modules/git/command.go

```diff
@@ -52,6 +52,14 @@ func NewCommand(args ...string) *Command {
 	}
 }
 
+// NewPureCommand creates and returns a new Git Command based on given command and arguments without global command args
+func NewPureCommand(args ...string) *Command {
+	return &Command{
+		name: GitExecutable,
+		args: args,
```

Comment:
We still need to run in the default locale otherwise tests will fail if git does not default to report its errors in English

Reply:
This is only a simple command with no args. All args includes global arguments will be given from function external.